### PR TITLE
[WNMGDS-855] Create github search utility

### PR DESCRIPTION
**Pull request by kimieann**
## Summary
The purpose of this PR is to create a utility we can use to search the Github API for which projects are using which design system versions.

## How to test
In the top right of CMS github, click your user icon > Settings.
[Choose Developer Settings in the left menu](https://github.com/settings/apps).
Choose personal access tokens.
Create a personal access token and write down the token.

- Pull down this branch
- cd to the utilities/findDesignVersion folder
- yarn install
- From the command line run `ts-node findDesignVersions.ts -- YOUR_PERSONAL_ACCESS_TOKEN`

It should print out repositories, file names, and design system versions.

Some of the respositories though match the search term, but don't have version numbers, example: 
image

Should we consider completely ignoring some repositories?

### CODE REVIEW

**Comment by line47** [comment]
I'm getting `bash: ts-node: command not found` when I try and run the final command

**Comment by kimieann** [comment]
> I'm getting `bash: ts-node: command not found` when I try and run the final command

Make sure to run yarn install from within the `utilities/findDesignVersion` directory, and also run the `ts-node` command from that directory.

If that doesn't work you can try running `yarn add ts-node directly`

**Comment by line47** [comment]
Hmmm, I tried the instruction you mentioned above and still get the issue `bash: ts-node: command not found` even after running `yarn add ts-node directly` and seeing that it installed

**Comment by line47** [comment]
This is working for me! 🎉

**Review by line47 — APPROVED** [review]

File: utilities/findDesignVersion/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+In order to gather the version currently being used for the design system packages across the CMS-WDS organization, findDesignVersions.ts utility was created.
+
+First, you'll need to create a personal access token through Github.
+
+In the top right of CMS github, click your user icon > Settings.
+Choose Developer Settings in the left menu.
+Choose personal access tokens.
```
Comment (line47) [inline utilities/findDesignVersion/README.md:7]:
```suggestion
```

File: utilities/findDesignVersion/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+In order to gather the version currently being used for the design system packages across the CMS-WDS organization, findDesignVersions.ts utility was created.
+
+First, you'll need to create a personal access token through Github.
+
+In the top right of CMS github, click your user icon > Settings.
+Choose Developer Settings in the left menu.
```
Comment (line47) [inline utilities/findDesignVersion/README.md:6]:
```suggestion
Choose Developer Settings in the left menu then [choose personal access tokens](https://github.cms.gov/settings/tokens).

```

**Review by nichia — APPROVED** [review]
The search utility works for me.

**Pull request merged**